docs/api-client/README.md

```diff
@@ -0,0 +1,929 @@
+| `sortBy` | `string` | **Required.** The way in which returned contracts are to be sorted: `creationdate`, `privatename`, or `title` |
+
+Example
+
+```
+
+```
+
+**Response**
+
+Returns an array of contracts.
+
+Example
+
+```
+[
+
+]
+```
+
+### sendContract
+
+Send a contract to other users.
+
+```
+POST /send/contract
+```
+
+**Headers:** `'Content-Type': 'application/x-www-form-urlencoded'`
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `readonlyEmails` | `Array<string>` | An array of users' emails who will have read-only access to the contract. |
+| `editEmails` | `Array<string>` | An array of users' emails who will have access to edit the contract.
+| `id` | `string` | **Required.** The ID of the contract to be sent.
+
+Example
+
+```
+
+```
+
+### changeContractAlias
+
+Change private name of contract.
+
+```
+GET /contract/alias/:contractId
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `contractId` | `string` | **Required.** The ID of the contract to be given a new private name. |
+| `newName` | `string` | **Required.** The new private name of the contract. |
+
+Example
+
+```
+
+```
+
+### stopContract
+
+Stop smart contract transactions that are pending for an executed contract.
+
+```
+GET /contract/stop/:id
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `id` | `string` | **Required.** The ID of the contract with the pending smart contract transactions. |
+
+Example
+
+```
+
+```
+
+### resumeContract
+
+Resume scheduled execution of smart contract transactions that were stopped for an executed contract.
+
+```
+GET /contract/resume/:id
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `id` | `string` | **Required.** The ID of the contract with the stopped smart contract transactions. |
+
+Example
+
+```
+
+```
+
+### uploadContractToGoogle
+
+Upload a contract to user's Google Drive.
+
+```
+GET /driveAuthPage/:id
+```
+
+**Headers:** `Origin: 'location.hostname'`, `'Access-Control-Allow-Origin': '*'`
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `id` | `string` | **Required.** The ID of the contract to be uploaded to Google Drive. |
+
+Example
+
+```
+
+```
+
+**Response**
+
+Returns `string` - the URL of the Google Drive API for redirecting.
+
+Example
+
+```
+
+```
+
+## User
+
+### getUserDetails
+
+Method used as part of `IdentityVariable` to get information about a user.
+
+```
+GET /user/details
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `email` | `string` | **Required.** The email of the user. |
+
+Example
+
+```
+
+```
+
+**Response**
+
+Returns an object containing information about a user, including `id`, `name`, `email`, and `identifiers`.
+
+Example
+
+```
+{
+
+}
+```
+
+### getStripeAccounts
```

Comment:
Let's leave this out until the Stripe integration is complete/merged.

Reply:
Agreed - removing for now.